client/src/index.jsx

```diff
@@ -19,7 +19,7 @@ class App extends React.Component {
   componentDidMount() {
     const bookId = window.location.pathname.split('/')[1] || 0;
 
-    fetch(`http://bookdescriptionservice-env.gp7avcyjnp.us-east-2.elasticbeanstalk.com/books/${bookId}`)
+    fetch(`/books/${bookId}`)
```

Comment:
Nice, totally forgot about branches for changes. I need to put my changes into a feature branch as well, good job!